extensions/dicom-microscopy/src/utils/loadAnnotation.js

```diff
@@ -0,0 +1,70 @@
+import getDicomWebClient from './dicomWebClient';
+
+export default function loadAnnotation({
```

Comment:
Can you give a couple of sentences that this loads the Ann bulk microscopy annotations, which are grouped by position ....

Reply:
Updated!